sdk/ai/azure-ai-projects/README.md

```diff
@@ -17,6 +17,10 @@ For example, get the inference endpoint URL and credentials associated with your
 | [SDK source code](https://aka.ms/azsdk/azure-ai-projects/python/code)
 | [AI Starter Template](https://aka.ms/azsdk/azure-ai-projects/python/ai-starter-template)
 
+## Reporting issues
+
+To report an issue with the client library, or request additional features, please open a GitHub issue [here](https://github.com/Azure/azure-sdk-for-python/issues). Mention the package name "azure-ai-projects" in the title or content.
+
```

Comment:
Unrelated change: Mention close to the top of the package README.md where to open issues. At the moment it's buried toward the bottom of the doc.